README.md

```diff
@@ -0,0 +1,42 @@
+<p align="center"><img src="/src/socialcard.png" alt="Social Card of FakerPHP"></p>
+
+# FakerPHP
+
+"Fake it 'till you make it"
+
+## Colors
+
+|                            |#hex                                                                                         |rgb()              |
+|---                         |---                                                                                          |---                |
+|![500](https://res.cloudinary.com/caneco/image/upload/c_scale,co_rgb:000000,e_colorize:100,f_png/v1/pallete.svg)|`#000000`|`rgb(0,0,0)`       |
+|![500](https://res.cloudinary.com/caneco/image/upload/c_scale,co_rgb:FFFFFF,e_colorize:100,f_png/v1/pallete.svg)|`#FFFFFF`|`rgb(255,255,255)` |
+
+## Requirements
+
+- A screen or a printer
+
+## Installation
+
+- Open the file
+- *Right-click* on the image
+- Choose **"Save image as…"** option
+
+## Maintainers
+
+**FakerPHP** logo is designed and maintained by [Caneco](https://twitter.com/caneco).
+
+## License
+
+All rights reserved, but with the following extra conditions:
+
+- It is **OK** to use the FakerPHP logo in the following cases:
+    - In marketing materials for technical events, e.g. meetups, hackathons, conferences and workshops that are related to Laravel.
```

Comment:
Do we need a reference to Laravel here?

Reply:
Agree, no need to have specifics here

Reply:
Oops… really sorry about that. #BadCopyPaste.

Should be changed to:

> … conferences and workshops that are related to **PHP or FakerPHP**.

Do you agree?

Reply:
Agreed